games-vue-client/src/components/AuthChoice.vue

```diff
@@ -1,30 +1,59 @@
 <template>
   <v-container>
-    <h2>Choose authentication type</h2>
-    <v-btn
-      color="info"
-      @click="authenticate('github')"
+    <h2>Sign in with</h2>
+    <v-card
+      class="d-flex justify-center mb-4"
+      flat
+      tile
     >
-      Login with Github
-    </v-btn>
-    <v-btn
-      color="info"
-      @click="authenticate('google')"
-    >
-      Login with Google
-    </v-btn>
-    <v-btn
-      color="info"
-      @click="authenticateGuest()"
-    >
-      Login as Guest
-    </v-btn>
-    <!--
+      <v-card
+        outlined
+        class="d-flex flex-column pa-8"
+      >
+        <v-btn
+          large
+          color="info"
+          class="mb-4 auth-btn"
+          @click="authenticate('github')"
+        >
+          Github
+        </v-btn>
+
+        <div
+          id="google-btn"
+          class="mb-4 auth-btn"
+          @click="authenticate('google')"
+        >
+          <span class="icon" />
+          <span class="buttonText">Google</span>
```

Comment:
Shouldn't this text be "Sing in with Google" according to their style guide?

Reply:
I was hoping that the title for the login options would cover the option of having the text "adjacent". However, I wonder if a title is adjacent enough.

"You can use the word "Google" by itself in the button if it is accompanied by adjacent text that makes it clear what the action is."
- https://developers.google.com/identity/branding-guidelines#matching

Reply:
I think it's adjacent enough for the moment, the option would be to have "Sign in" on every button, which could also work I guess (and is similar to how I've done it on https://play.minesweeperflags.net)